src/components/LearnMore.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { Container } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+import Button from '@material-ui/core/Button';
+import logopic from '../img/blank.png';
+
+export default function LearnMore() {
+  return (
+    <Container style={{ textAlign: 'left', paddingTop: '50px' }}>
```

Comment:
`textAlign` shouldn't need to be set. Try inspecting this element in the browser and figuring what's changing the text alignment and fix that instead.

`paddingTop` is not necessary here either

Reply:
Ok got it figured out how to resolve this
& also paddingTop is removed but there might need to be space above once the component `ScholarshipMadeSimple` is above